docs/docs/guides/labs/observability-update/index.md

```diff
@@ -6,21 +6,54 @@ sidebar_position: 100
 
-### How to join and enable the beta
+### How to join the early access program
 
-The full suite of new features is available in limited early access for Dagster+ users. Sign up to join the early access program [here](https://dagster.io/summer).
+The full suite of new features is available in limited early access for Dagster+ users. To join the early access program, sign up on the [Dagster+ Summer Update page](https://dagster.io/summer).
 
 Once you are a member of the early access program, you can enable or disable the new experiences in your user settings, via the "New health & observability UIs" setting.
 
 ## Features
 
-- Asset health reporting
-- [Freshness monitoring](/guides/labs/observability-update/freshness)
-- [Real-time insights](/guides/labs/observability-update/insights)
-- Customizable catalog
+### New homepage
+
+Get a holistic view of your data platform’s health from the moment you log in. The redesigned Dagster+ homepage keeps your most important signals front and center so you can track key metrics in real time, surface issues quickly, and stay focused on what matters most.
+
+![New Dagster+ homepage](/images/guides/labs/observability-update/new-dagster-plus-homepage.png)
+
+### Freshness policies
+
+Ensure your most critical data stays fresh and trustworthy so teams can make timely, confident decisions without second-guessing the source. For more information, see the [Freshness policies guide](/guides/labs/observability-update/freshness).
+
+### Asset health reporting
+
+Quickly identify which datasets are performing well and which need attention with intuitive health indicators that highlight data quality and platform reliability in real time. For more information, see the [Asset health reporting guide](/guides/labs/observability-update/asset-health).
+
+### Real-time Insights
+
+Unlock a comprehensive, dashboard-style view of your data platform’s health. The new Insights experience surfaces critical metrics, like success rate, freshness hit rate, and time to resolution for any selection of assets or jobs – all in real time. For more information, see the [Real-time Insights guide](/guides/labs/observability-update/insights).
+
+### Custom asset catalog dashboards
+
+Give teams a focused view of the data that matters most. Create flexible, shareable dashboards organized by asset, owner, tag, or business domain so every stakeholder can monitor performance and act with clarity. For more information, see the [Asset catalog with custom selections guide](/guides/labs/observability-update/asset-catalog).
+
+### Asset facets
+
+With asset facets, you can customize asset views to surface the most relevant metadata, like ownership, health, freshness, and automation signals.
```

Comment:
Even though the content here is pretty minimal, I think it should go in a separate page for two reasons:
- it visually takes up a lot of space in this page because of the images
- this actually isn't inherently tied to the Observe beta. Everyone has access to this now.

I think facets subpage can be placed outside the Observability update hub, and also get linked to from the "Global Asset Lineage" section in https://docs.dagster.io/guides/operate/webserver